doc/filters/format_datetime.rst

```diff
@@ -24,7 +24,9 @@ You can tweak the output for the date part and the time part:
     {# mercredi 7 août 2019 23:39:12 UTC #}
     {{ '2019-08-07 23:39:12'|format_datetime('full', 'full', locale='fr') }}
 
-Supported values are: ``none``, ``short``, ``medium``, ``long``, and ``full``.
+Supported values are: ``none``, ``short``, ``medium``, ``long``, and ``full``. ``relative_short``, ``relative_medium``,
+``relative_long``, and ``relative_full`` are also supported when running on PHP 8.0 and superior or when using a
+polyfill that define the ``IntlDateFormatter::RELATIVE_*`` constants and associated behavior.
```

Comment:
The support part should be extract in a `.. versionadded::` directive as done elsewhere in the docs.

Reply:
Wich version should I use as a target? It depends on the running PHP version (8.0) or the usage of a polyfill. Others `.. versionadded::` targets a Twig release. Maybe `3.6` with a proper explanation in the block?